clients/java-logger/shell/src/test/java/com/abcxyz/lumberjack/loggingshell/LoggingControllerTest.java

```diff
@@ -77,48 +77,23 @@ void loggingWithoutTraceIdCausesBadRequestError() throws Exception {
   }
 
   @Test
-  void loggingWithTraceIdReturnsOk() throws Exception {
-    mockMvc
-        .perform(GET_REQUEST_BUILDER_WITH_EMAIL_AND_TRACE_ID_AND_JVS_TOKEN)
-        .andExpect(status().isOk());
```

Comment:
When you remove this one, did you verify it in other test cases that the status is ok?

Reply:
I think the removed thing is duplicated to the one in line86, so if `loggingWithTraceIdAndCustomInfoAndAssertResult` passes, the status will also be checked.